lib/src/html_generator.dart

```diff
@@ -547,8 +547,13 @@ class ClassTemplateData extends TemplateData {
       '${clazz.name} ${clazz.kind} - ${_library.name} library - Dart API';
   String get metaDescription =>
       'API docs for the ${clazz.name} ${clazz.kind} from the ${_library.name} library, for the Dart programming language.';
+  String get _fullkind {
```

Comment:
consider for later to move this to the model, and that way we can add a test for it :)